_includes/service_card.html

```diff
@@ -1,11 +1,11 @@
 {% if include.reversed %}
-  {% assign margin = 'mr-12' %}
+  {% assign margin = 'md:mr-12' %}
 {% else %}
-  {% assign margin = 'ml-12' %}
+  {% assign margin = 'md:ml-12' %}
 {% endif %}
 
 {% capture image %}
-  <img class="rounded max-w-md" src="{{ include.image | prepend:site.baseurl }}" />
+  <img class="rounded mb-5 md:mb-0 md:max-w-md" src="{{ include.image | prepend:site.baseurl }}" />
```

Comment:
In here, `mb-5` will be applied to all sizes but `md:mb-0` means from medium screens onwards we'll override that with `mb-0`